parsing/lexer.mll

```diff
@@ -290,6 +290,9 @@ let identchar_latin1 =
   ['A'-'Z' 'a'-'z' '_' '\192'-'\214' '\216'-'\246' '\248'-'\255' '\'' '0'-'9']
 let symbolchar =
   ['!' '$' '%' '&' '*' '+' '-' '.' '/' ':' '<' '=' '>' '?' '@' '^' '|' '~']
+let quoted_string_id_char =
+  lowercase |
+      ['!' '$' '%' '&' '*' '+' '-' '.' '/' ':' '<' '=' '>' '?' '@' '^' '~']
```

Comment:
This could be more clearly defined as:
```ocaml
let common_symbolchar =
  ['!' '$' '%' '&' '*' '+' '-' '.' '/' ':' '<' '=' '>' '?' '@' '^' '~']
let symbolchar =
  common_symbolchar | '|'
let quoted_string_id_char =
  lowercase | common_symbol_char
```